sql/testdata/table

```diff
@@ -282,6 +282,12 @@ CREATE TABLE test.dupe_named_constraints (
   INDEX (name)
 )
 
+statement error duplicate constraint name: "one"
+CREATE TABLE test.dupe_named_constraints (
+  id        INT CONSTRAINT pk PRIMARY KEY,
+  title     VARCHAR CONSTRAINT one CHECK (1>1) CONSTRAINT one CHECK (1<1)
+)
+
```

Comment:
Might be worth including tests for the other usage errors now explicitly disallowed (multiple defaults, references, etc).